src/stories/Editor.stories.tsx

```diff
@@ -137,6 +137,19 @@ ToggleDisabledProp.parameters = {
   controls: { hideNoControlsWarning: true },
 };
 
+export const PlaceholderProp: Story = () => {
+  return (
+    <div>
+      <Editor
+        apiKey={apiKey}
+        placeholder="This is some placeholder text that works like a placeholder"
+      />
```

Comment:
I was wondering why you want this feature when you can already do:
```jsx
<Editor init={{ placeholder: 'This is placeholder text' }} />
```

Currently the `placeholder` can't be updated after `init` anyway (though that may change with the options API in TinyMCE 6) so I'm worried that this currently adds maintenance burden without any real advantage over the existing method.

If you have a use-case then I'm happy to merge this. I just don't want to be adding props for things that can already be done without an obvious upside.